MonMaraicher/Features/FarmerDetail/FarmerDetailsView.swift

```diff
@@ -36,20 +37,22 @@ struct FarmerDetailsView: View {
                         .clipShape(.rect(cornerRadius: 16))
                         .shadow(radius: 2)
                         .allowsHitTesting(false)
```

Comment:
Why did you add this modifier?

Reply:
To avoid having a bad experience when the user scrolls and touches the mini map

Reply:
Ok makes sense